ansible/roles/postgresql/tasks/main.yml

```diff
@@ -90,9 +90,9 @@
   tags: postgresql_conf
   notify: Reload postgres
 
-- name: start postgresql
```

Comment:
Why did this change from start to restart? I think that this will get included everytime we run a postgres command which means that postgres may restart even if that's unecessary.

Also if it's intended to restart should we change the register below to `restart_postgresql` ?

Reply:
I had to change it to restart because on the first pass, I believe by this point it was already started, so `state=started` did nothing; however since the configuration files just changed above, it's important that postgresql is restarted, or else the following command fails.

I see what you're saying about this forcing a restart. Maybe if I just make sure that it gets reloaded/restarted if any of the files change, this will still work and not have the same negative effect. (The `notify` technique above doesn't cut it because that only does it after the playbook is over. Maybe I should just remove those as well then.)

Reply:
You can force the handlers to run with:
```
- meta: flush_handlers
```

Reply:
Oh, that's great! Thanks, Simon.

Reply:
https://github.com/dimagi/commcare-cloud/pull/1467